docs/guide/search.md

```diff
@@ -422,6 +422,59 @@ ezpublish.search.legacy.gateway.sort_clause_handler.location.depth:
 !!! note "See also"
 
     See also [Symfony documentation about Service Container](http://symfony.com/doc/current/book/service_container.html#service-parameters) for passing parameters.
+    
+### Search using custom Field Criterion [REST]
+      
+REST search can be performed via `POST /views` using custom `FieldCriterion`. This allows you to build custom content logic queries with nested logical operators OR/AND/NOT. This functionality works for all custom fields. 
+
```

Comment:
>This functionality works for all custom fields. 

Don't know if this is the case, but this search don't works for all the fields (like rich text for example.)